barman/server.py

```diff
@@ -797,10 +823,64 @@ def check_postgres(self, check_strategy):
                     hint="please set it to a higher level than 'minimal'",
                 )
 
+    def _check_has_monitoring_privileges(
+        self, check_strategy, remote_status, suffix=None
+    ):
+        check_name = "has monitoring privileges"
+        if suffix is not None:
+            check_name = f"{check_name} ({suffix})"
```

Comment:
```suggestion
        check_name = "has monitoring privileges" + ("" if suffix is None else f" ({suffix})")
```

Same comment in regards to single attribution.